src/Redistribution.sol

```diff
@@ -115,6 +92,7 @@ contract Redistribution is AccessControl, Pausable {
         bytes signature;
         bytes32 chunkAddr;
         bytes32 postageId;
+        uint64 chunkSpan;
```

Comment:
yes, this was mistake. will redo it now as this is really chatoic.

Reply:
I reordered struct because that was part of memory packing optimization.
I do not have any idea what you are referring to as breaking data, don't think this is the case, please provide more info here.